_posts/2022-05-13-survey-2022.md

```diff
@@ -0,0 +1,47 @@
+---
+layout: post
+title: "Strimzi survey: we want to hear from you!"
+date: 2021-05-13
+author: paolo_patierno
+---
+
+As every year, with the help of the CNCF, we run a survey of the Strimzi community.
+The main purpose is about hearing from the users in order to know how they are using Strimzi.
+This is really useful to the maintainers to make better decisions in terms of new features and define the roadmap of the project itself.
+
+It is a really short survey with a few questions but it represents a huge help for us.
+It would be great if you could take the time for it.
+
+<!--more-->
+
+## What we are interested in?
+
+We are interested to know what is your usage of Strimzi, if just for development, evaluation or already in production.
+In order to provide better support and backward compatibility, it is important to get the most used Kubernetes versions.
+Kubernetes API changes and deprecations can have an impact on the Strimzi project as well.
+
+Upgrading a Strimzi cluster can be done at different times by different users.
+On every new operator release, when new valuable features are added or when a new Kafka version is supported. What is your choice?
+
+One of the most recent features was the Kafka Connect Build.
```

Comment:
Can we link to a blog or video about this?

Reply:
Yeah, I added a link to this one https://strimzi.io/blog/2021/03/29/connector-build/